framework/src/st_things/things_stack/utils/things_string.c

```diff
@@ -81,7 +81,7 @@ int things_string_compare(const char *src, const char *dst)
 		return -1;
 	}
 
-	return strncmp(src, dst, dstLen);
+	return strncmp(src, dst, dstLen + 1);
 }
```

Comment:
The main concept of this commit is "secure coding" on strncmp.
which means, it's to avoid data abort during running strlen().

things_string_compare() function should be re-implemented without using strlen